packages/cmrjs/cmr-utils.js

```diff
@@ -270,16 +270,36 @@ async function bucketsConfigDefaults() {
   return bucketConfig(process.env.system_bucket, process.env.stackName);
 }
 
+/**
+ * Build and return an S3 Credentials Object for adding to CMR onlineAccessUrls
+ *
+ * @param {string} s3CredsUrl - full url pointing to the s3 credential distribution api
+ * @returns {Object} Object with attributes required for adding an onlineAccessUrl
+ */
+function getS3CredentialsObject(s3CredsUrl) {
+  return {
+    URL: s3CredsUrl,
+    URLDescription: 'api endpoint to retrieve temporary credentials valid for same-region direct s3 access',
+    Description: 'api endpoint to retrieve temporary credentials valid for same-region direct s3 access',
+    Type: 'VIEW RELATED INFORMATION'
+  };
+}
+
 /**
  * Construct a list of online access urls.
  *
- * @param {Array<Object>} files - array of file objects
- * @param {string} distEndpoint - distribution endpoint from config
- * @param {BucketsConfig} buckets -  Class instance
+ * @param {Object} params - input parameters
+ * @param {Array<Object>} params.files - array of file objects
+ * @param {string} params.distEndpoint - distribution endpoint from config
+ * @param {BucketsConfig} params.buckets -  Class instance
+ * @param {string} params.s3CredsEndpoint - optional s3credentials endpoint name
  * @returns {Array<{URL: string, URLDescription: string}>}
  *   returns the list of online access url objects
  */
-function constructOnlineAccessUrls(files, distEndpoint, buckets) {
+function constructOnlineAccessUrls(params) {
```

Comment:
I think this is mostly stylistic, but I would probably just destructure the parameters right in the function signature, which I think simplifies having default values for the parameters as well.

```suggestion
function constructOnlineAccessUrls({
  files,
  distEndpoint,
  buckets,
  s3CredsEndpoint = 's3credentials'
}) {
```